include/ceed/ceed.h

```diff
@@ -508,6 +508,8 @@ CEED_EXTERN int CeedOperatorGetNumElements(CeedOperator op, CeedInt *num_elem);
 CEED_EXTERN int CeedOperatorGetNumQuadraturePoints(CeedOperator op, CeedInt *num_qpts);
 CEED_EXTERN int CeedOperatorGetFlopsEstimate(CeedOperator op, CeedSize *flops);
 CEED_EXTERN int CeedOperatorContextGetFieldLabel(CeedOperator op, const char *field_name, CeedContextFieldLabel *field_label);
+CEED_EXTERN int CeedOperatorContextGetDouble(CeedOperator op, CeedContextFieldLabel field_label, size_t *num_values, const double **values);
+CEED_EXTERN int CeedOperatorContextGetInt32(CeedOperator op, CeedContextFieldLabel field_label, size_t *num_values, const int **values);
```

Comment:
Any reason to return `size_t *num_values`? The user would already need to ensure that `*values` is of the correct size, so they probably already know `num_values` (or could find out from the label itself).

Reply:
I nearly always send the array length with the array so that each function call can stand alone

Reply:
Ah, didn't see that `values` was a double pointer. So my latter point is invalid (user doesn't have to know the length of the array before the call).